test/test_experiment_parameters.py

```diff
@@ -21,6 +21,21 @@ def test_describe(self):
                 "type": "float"
             })
 
+    def test_evaluate_default(self):
+        def placeholder_get_default(key, default):
```

Comment:
As per the parameter name, this function is supposed to be `get_dataset`, not `get_default`. The first parameter is also supposed ot be a string as per the type annotations (although this is not currently enforced at execution time), so the tests are weirdly backwards.

Maybe something like
```py
def mock_get_dataset(key, default=None):
  return {"baz": 42}[key]
```
which would also check that the key is passed correctly, and below `IntParam("foo", "bar", "dataset('baz')")`?

Reply:
Other than the previous comment, implemented in previous commit.

Reply:
I was just referring to the dataset() argument order in the previous tests.